pom.xml

```diff
@@ -11,6 +11,10 @@
 
 	<name>holeshot</name>
 
+	<prerequisites>
```

Comment:
This is a good check to have, but it isn't used with maven 3 anymore (See https://maven.apache.org/pom.html#Prerequisites ).   This will be caught by someone trying to run with maven 2, so we should probably leave it in, but we should also add a section for requireMavenVersion on the maven-enforcer-plugin. (https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html)